db/migrate/20241203150351_create_one_login_auths.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOneLoginAuths < ActiveRecord::Migration[8.0]
+  def change
+    create_table :one_login_auths do |t|
+      t.string :email, null: false
```

Comment:
Pedantry warning: elsewhere in the code we use 'email_address' rather than 'email'. Can we be consistent?